packages/console-app2/src/components/ResourceList.tsx

```diff
@@ -2,14 +2,13 @@
 // Copyright 2021 DXOS.org
 //
 
-import React, { useEffect, useState } from 'react';
-import { List } from 'semantic-ui-react'
+import { useEffect, useState } from 'react';
+import { List } from 'semantic-ui-react';
```

Comment:
Please revert -- must use material-ui. Please follow existing patterns. See existing console-app.

Reply:
Sorry, I only adopted linter suggestions here. No change from material-ui was made. So I don't understand the issue.

We plan to use material-ui, of course, but this patch is only about cleanup. Would that be OK if we will use material-ui with the following changes?

Reply:
@richburdon -> material-ui is yet to come (a next step).

This PR does not add nor remove semantic UI. The code you refer to was added in https://github.com/dxos/console/commit/cd658886e07d5ecd3d592ff0abca36a6ffad13e3 by you @richburdon and @egorgripasov :)

@vitaliyworks just improves imports and adds `;` :)

Yet again -> we're going to follow the org patterns and recipes but in the next PRs